versions.tf

```diff
@@ -19,16 +19,31 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 4.66"
```

Comment:
It would be good to keep the `< 5.0` constraint as 5.0 may contain breaking changes.

Reply:
Done.